.github/workflows/maven-build.yml

```diff
@@ -45,7 +46,32 @@ jobs:
         CI_SERVICE: GITHUB
         TZ: Europe/Amsterdam
         JAVA_OPTS: "-Xms1G -XX:+UseParallelGC"
-      run: mvn -B -V -T1 -Pibissource,codecoverage -Dmaven.javadoc.skip=true package
+      run: mvn -B -V -T1 -Pibissource,codecoverage -Dmaven.javadoc.skip=true package install
+      working-directory: frank-doc
 
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v1
+
+    - name: Read Frank!Doc version
+      run: bash frank-doc/.github/workflows/getFrankDocVersion.sh
+
+    - name: Checkout Frank!Framework
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+        repository: ibissource/iaf
+        path: iaf
+
+    - name: Build Frank!Framework
+      env:
+        CI_SERVICE: GITHUB
+        TZ: Europe/Amsterdam
+        JAVA_OPTS: "-Xms1G -XX:+UseParallelGC"
+      run: mvn -B -V -T1 -PattachFrankDoc -DskipTests -Dfrankdoc.version=$(cat ../frankDocVersion.txt) process-sources
+      working-directory: iaf
```

Comment:
Kan je niet beter het framework headless draaien?

Reply:
Op die manier kan je niet testen of FrankConfig-strict.xsd een valide configuratie kan valideren.

Zie verder issue https://github.com/ibissource/frank-doc/issues/106.